pkg/compute/assemblyscript.go

```diff
@@ -52,7 +53,7 @@ func (a AssemblyScript) Verify(out io.Writer) error {
 		return fmt.Errorf("getting package.json path: %w", err)
 	}
 
-	if !common.FileExists(fpath) {
+	if !filesystem.FileExists(fpath) {
```

Comment:
I wanted to use an existing function `MakeDirectoryIfNotExists` but I couldn't because of a cyclic import issue, so I needed to move those functions into a separate module (as `common` appeared to be containing too much miscellaneous behaviour).